setup/security/authentication/oauth/index.md

```diff
@@ -95,63 +90,28 @@ userInfoMapping:
   username: user
 ```
 
-#### Enable your custom provider
-
-Configure your custom OAuth Provider in Halyard
-
-```
-CLIENT_ID=myClientId
-CLIENT_SECRET=myClientSecret
-
-hal config security authn oauth2 edit \
-  --client-id $CLIENT_ID \
-  --client-secret $CLIENT_SECRET
-```
-
-Enable the oauth2 Provider in Halyard
-
-```
-hal config security authn oauth2 enable
-```
-
-
-## Network architecture and SSL termination
-
-During the OAuth [workflow](/reference/architecture/authz_authn/authentication/#workflow), Gate makes an intelligent 
-guess on how to assemble a URI to
-itself, called the **`redirect_uri`**. Sometimes this guess is wrong when Spinnaker is deployed
-in concert with other networking components, such as an SSL-terminating load balancer, or in the
-case of the [Quickstart](/setup/quickstart) images, a fronting Apache instance.
-
-To manually set the `redirect_uri` for Gate, use the following `hal` command:
-
-```bash
-hal config security authn oauth2 edit --pre-established-redirect-uri https://my-real-gate-address.com:8084/login
-```
-
-> Be sure to include the `/login` suffix at the end of the `--pre-established-redirect-uri` flag!
+## Restricting access based on User Info
 
-Additionally, some configurations make it necessary to "unwind" external proxy instances. This makes the request to Gate
-look like the original request to the outer-most proxy. Add this to your `gate-local.yml` file in your Halyard
-[custom profile](/reference/halyard/custom/#custom-profiles):
+User access can be restricted further based on the user info from an OAuth ID token. This
+requirement is set via the `security.authn.oauth2.userInfoRequirements` field, which
+is a map of key/value pairs. The values are interpreted as regular expressions if
+if they start and end with '/'. This enables us to restrict user login to specific
```

Comment:
"us"?

Reply:
I'd moved this from below, so that's pre-existing, but agree it's unclear who "us" is; changed to use the gerund form which eliminates an ambiguous subject.